File: app/src/main/java/md/fusionworks/lifehack/ui/sales/model/SaleCategoryModel.java
package md.fusionworks.lifehack.ui.sales.model;

/**
 * Created by ungvas on 2/24/16.
 */
public class SaleCategoryModel {

  public int id;
  public String nameRu;
  public String nameRo;
  public boolean isHeader;
  public String header;

  public SaleCategoryModel(int id, String nameRu, String nameRo) {
    this.id = id;
    this.nameRu = nameRu;
    this.nameRo = nameRo;
  }

  public SaleCategoryModel(int id, String nameRu, String nameRo, boolean isHeader, String header) {
    this.id = id;
    this.nameRu = nameRu;
    this.nameRo = nameRo;
    this.isHeader = isHeader;
    this.header = header;
  }
}

File: app/src/main/java/md/fusionworks/lifehack/sales/model/SaleCategoryModel.kt
package md.fusionworks.lifehack.sales.model

/**
 * Created by ungvas on 2/24/16.
 */
class SaleCategoryModel {

  var id: Int = 0
  var nameRu: String
  var nameRo: String
  var isHeader: Boolean = false
  var header: String = ""

  constructor(id: Int, nameRu: String, nameRo: String) {
    this.id = id
    this.nameRu = nameRu
    this.nameRo = nameRo
  }

  constructor(id: Int, nameRu: String, nameRo: String, isHeader: Boolean, header: String) {
    this.id = id
    this.nameRu = nameRu
    this.nameRo = nameRo
    this.isHeader = isHeader
    this.header = header
  }
}
